### PR TITLE
Pass through reaction_added messages

### PR DESCRIPTION
**Pull request by mbland**
Just sends `reaction_added` messages straight through without any special treatment. Used to implement behavior needed by 18F/hubot-slack-github-issues.

I understand that 2.0 is on the way soon, and #59 is also still open, so no hard feelings if this gets closed without merging.

cc: @afeld @ccostino

### CODE REVIEW

**Comment by l12s** [comment]
Ok, first off, I apologize for how long it's taken us to resolve this PR.

I just pushed the 2.0.0 release for this library, which is a ground up refactor with the aim of building a lib that both Slack and our external contributors can easily work with and improve.

That release now passes through all messages from the [RTM API by default](https://github.com/slackhq/node-slack-client/blob/03743068caa535f15a636da164953f402d1899c5/lib/clients/rtm/client.js#L356) as well as providing [handlers for reactions](https://github.com/slackhq/node-slack-client/blob/03743068caa535f15a636da164953f402d1899c5/lib/data-store/message-handlers/reactions.js) against objects in the in-memory data store. Hopefully that gives you what you need, but if not, please raise an issue or create a new PR.

**Comment by mbland** [comment]
Thanks, @l12s! Any idea when this upgraded client will get integrated into slackhq/hubot-slack?

**Comment by l12s** [comment]
I've added an issue to track this: https://github.com/slackhq/node-slack-client/issues/141

TLDR is I'd like the `>= 2.0.x` release to be in use for at least 2-4 weeks to shake out any bugs and then I'll talk to the folks who are running other OSS projects (botkit, hubot etc.) about what a transition plan should look like.
